docs/pages/machine-id/architecture.mdx

```diff
@@ -0,0 +1,154 @@
+---
+title: Machine ID Architecture
+description: How Teleport Machine ID works.
+---
+
+This section provides an overview of Teleport Machine ID's inner workings.
+
+A more in-depth specification of the workings of Machine ID can be found in
+[the Request For Discussion.](https://github.com/gravitational/teleport/blob/master/rfd/0064-bot-for-cert-renewals.md)
+
+## Bot creation
+
+When creating a bot via `tctl`, up to three resources are created:
```

Comment:
I think this sentence would be a lot clearer in the active voice. What is doing the creating? When does this creation take place? What creates the resources?

Also, I think we should assume that a reader with any level of familiarity with Machine ID could visit this page. We should either begin with more fundamental information about Machine ID or explicitly make reading the Introduction/Getting Started pages a prerequisite for reading this page.

As it stands now, a reader who is just getting acquainted with Machine ID won't understand what "Bot creation" or "When creating a bot via `tctl`" mean.

Reply:
I would much rather make reading the introduction a pre-requisite of this page, otherwise, this architecture guides conciseness and usefulness to a more aware user is going to be hampered by constant re-explanation of terms a user with a cursory knowledge of Machine ID should understand.